Utils/strUtils.py

```diff
@@ -0,0 +1,56 @@
+# Function to display string length #
+def disStrLength(str1 : str)-> int :
```

Comment:
not space after parameter, should have space after colon only

Reply:
don't use prebuild method like len(), use for loop for calculating the string lenth